src/core/datatypes/trees/Clade.cpp

```diff
@@ -5,24 +5,22 @@
 #include <string>
 #include <utility>
 
+#include "StringUtilities.h"
 #include "RbVectorUtilities.h"
 #include "RbException.h"
 
 using namespace RevBayesCore;
 
+using std::set;
+using std::string;
+using std::vector;
 
 /**
  * Default constructor required by the revlanguage code.
  * We use an empty string and an age of 0.0 for
  * this default object.
  */
-Clade::Clade( void ) :
-    age( 0.0 ),
-    clade_name(""),
-    num_missing( 0 ),
-    taxa(),
-    is_negative_constraint(false),
-    is_optional_match(false)
+Clade::Clade( void )
```

Comment:
why is it better to not initialize the values with specific defaults? I think that's better style because it's more explicit.

Reply:
They actually are initialized with specific defaults - but in the class declaration instead.
Then each constructor can mention which defaults are DIFFERENT from the global defaults in the class declaration.